### PR TITLE
update sig and example for lift/liftN function

### PR DESCRIPTION
**Pull request by running-grass**

### CODE REVIEW

File: source/liftN.js [hunk 1]
```diff
@@ -13,14 +13,21 @@ import map from './map';
  * @memberOf R
  * @since v0.7.0
  * @category Function
- * @sig Number -> (*... -> *) -> ([*]... -> [*])
+ * @sig Number -> (a, b, c, ... -> z) -> ([a], [b], [c], ... -> [z])
```
Comment (CrossEye) [inline source/liftN.js]: Shouldn't this  one end at `n`, not `z`?

**Comment by leo19920823** [comment]
@CrossEye  I have fixed it

**Comment by CrossEye** [comment]
I'm sorry, I may have been too terse.  I meant that for `liftN`, perhaps instead of `a, b, c, ..., z`, we should always have `a, b, c, ..., n`.  It should be clear that the `n` here is related to the one in `liftN`, the first parameter.

**Comment by leo19920823** [comment]
@CrossEye  But , If n is less than the number of parameters of the function, What will happen? crash?

**Comment by CrossEye** [comment]
Eventually handled in #3109

**Pull request closed without merging**